consensus/view_change.go

```diff
@@ -163,13 +167,13 @@ func (consensus *Consensus) startViewChange(viewID uint64) {
 	consensus.consensusTimeout[timeoutBootstrap].Stop()
 	consensus.current.SetMode(ViewChanging)
 	consensus.SetViewChangingID(viewID)
-	consensus.LeaderPubKey = consensus.GetNextLeaderKey()
+	consensus.NextLeaderPubKey = consensus.GetNextLeaderKey(viewID)
```

Comment:
This NextLeaderPubKey field is only used in here. Don't need to be a struct variable. it can just be a local variable.

Reply:
Fixed, can you review again.